src/jit/gentree.h

```diff
@@ -2399,12 +2399,23 @@ struct ReturnTypeDesc
 public:
     ReturnTypeDesc()
     {
+#ifdef _TARGET_X86_
+        m_regType0 = TYP_INT;
+        m_regType1 = TYP_INT;
+
+#ifdef DEBUG
+        m_inited = true;
+#endif
+
+#else
```

Comment:
Since the plan is to universally use a ReturnTypeDesc on calls (even single-reg), I think you should use the same constructor behavior, and then initialize it.